src/python/pants/backend/docker/goals/package_image.py

```diff
@@ -250,10 +255,18 @@ async def build_docker_image(
         ),
     )
 
+    image_tags_requests = union_membership.get(DockerImageTagsRequest)
+    additional_image_tags = await MultiGet(
+        Get(DockerImageTags, DockerImageTagsRequest, image_tags_request_cls(wrapped_target.target))
```

Comment:
Confirming the API decision that you can have multiple plugins operate on the same target? That makes sense to me.